metpy/calc/kinematics.py

```diff
@@ -432,3 +433,109 @@ def geostrophic_wind(heights, f, dx, dy):
     grad = _gradient(heights, *deltas)
     dy, dx = grad[-2:]  # Throw away unused gradient components
     return -norm_factor * dy, norm_factor * dx
+
+
+@exporter.export
+@check_units('[speed]', '[speed]', '[pressure]', '[length]',
+             '[length]', '[length]', '[speed]', '[speed]')
+def storm_relative_helicity(u, v, p, srh_top, hgt, srh_bottom=0, storm_u=0 * units('m/s'),
```

Comment:
I would drop the `srh_` prefix from top/bottom--they don't really add any information.

I think the order of the first 6 parameters should be:

u, v, p, height, top, bottom

You should also add units to the default value for bottom.